app/assets/stylesheets/utilities.scss

```diff
@@ -0,0 +1,51 @@
+//buttons
+.button-primary {
```

Comment:
Are you overriding bootstrap class here ?

Reply:
No, the bootstrap one is btn-primary. It is the class I made for our custom buttons; so that then it is just adding this class for all other pages without having additional styles

Reply:
may be `components.scss` ?

Reply:
Done